### PR TITLE
linking between pages

### PR DESCRIPTION
**Pull request by mewiemers**
don't know why the canvas is pushed also...

### CODE REVIEW

File: components/loginscreen/Loginscreen.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ function Logo() {
     <div className={styles.background}>
       <h1 className={styles.headline}>Meet Albert</h1>
       <Input />
-      <Button label="Discover" />
+      <Button label="Discover" link={"/introduce"} />
```
Comment (mrana17) [inline components/loginscreen/Loginscreen.tsx:11]: I'm not pretty sure but I remember something that we are supposed to use the <Link element or am I mixing something up??

**Review by mrana17 — APPROVED** [review]
Lgtm 🚀

**Review by DNaunin — APPROVED** [review]
Great Job! 🐨

**Review by phlgr — APPROVED** [review]
Gj 🚀

**Pull request merged**